src/HasteModuleLoader/HasteModuleLoader.js

```diff
@@ -113,6 +112,9 @@ function _getCacheFilePath(config) {
 
 function Loader(config, environment, resourceMap) {
   this._config = config;
+  this._CoverageCollector = require(
+    config.testCollector || '../CoverageCollector'
```

Comment:
(Oh it looks like you already added a default -- but it doesn't match what you have here... Did you mean to delete this?)